src/test/java/com/dampcake/bencode/BencodeOutputStreamTest.java

```diff
@@ -140,7 +141,7 @@ public void run() throws Exception {
     @Test
     @SuppressWarnings("unchecked")
     public void testWriteDictionary() throws Exception {
-        instance.writeDictionary(new HashMap<Object, Object>() {{
+        instance.writeDictionary(new TreeMap<Object, Object>() {{
```

Comment:
I think it makes more sense to use a `LinkedHashMap` and modify the expectation.
Will probably make the tests a little easier to read too!